pkg/parse/tests/numbers_format.sysl

```diff
@@ -0,0 +1,20 @@
+Numbers:
+    !type A:
+        aAttr0 <: int
+        aAttr1 <: float
+
+    !type B:
+        bAttr0 <: int32
+        bAttr1 <: int32(4)
```

Comment:
Only include the following types in your test:
```
int
int32
int64
float
float32
float64
```

Types like `int32(4)` currently parse but are unsupported. I've [added](https://github.com/anz-bank/sysl/issues/1044) a ticket to deprecate unsupported values.